coops-ui/package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "vue-cli-service lint",
     "lint:nofix": "vue-cli-service lint --no-fix",
     "test:e2e": "vue-cli-service test:e2e",
-    "test:unit": "vue-cli-service test:unit --testPathPattern=.",
```

Comment:
Please restore this. It provides an easy-to-fill-out pattern for a developer to use to test a specific component.